install/install.go

```diff
@@ -126,11 +209,97 @@ func printMessage(msg string, flag bool) int {
 	return 0
 }
 
-func checkPods(c *k8s.Client, o Options) {
+func checkPods(c *k8s.Client, o Options, i bool) {
+	stime := time.Now()
+	otime := stime.Add(600 * time.Second)
+	cursor := [4]string{"|", "/", "—", "\\"}
+	// Check snitch completion only for install
+	if i {
+		for {
+			time.Sleep(900 * time.Millisecond)
+			pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app=kubearmor-snitch", FieldSelector: "status.phase==Succeeded"})
+			podno := len(pods.Items)
+			remaining := time.Until(otime).Round(time.Second)
+			fmt.Printf("\rℹ️\tWaiting for KubeArmor Snitch to run:  %s, estimated time remaining: %v", cursor[cursorcount], remaining)
```

Comment:
I don't it's a good idea to display this esitmated time. 
Let's just mention, this may take a couple of minutes.
Also this is not cleared up after it actually succeeds

![image](https://github.com/kubearmor/kubearmor-client/assets/47106543/a4f7a311-c6a0-42f1-bb79-07e24c9abf29)

Reply:
@daemon1024 does this look good?
Currently not able to figure out the exact of reason of  `\` residue
![Screenshot from 2024-03-12 11-32-33](https://github.com/kubearmor/kubearmor-client/assets/73490663/45548720-0348-46a5-b640-a87e8ec0ad54)

Reply:
![Screenshot from 2024-03-12 18-17-49](https://github.com/kubearmor/kubearmor-client/assets/73490663/3d185127-05a5-4aea-a61e-00c3defbe336)